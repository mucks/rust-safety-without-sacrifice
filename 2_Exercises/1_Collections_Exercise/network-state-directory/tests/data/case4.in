# Alliances and list all
MEMBER Zuitzerland 10 Zoe Citizen
MEMBER Prospera 11 Pax Founder
ALLIANCE ForestCity â†” Zuitzerland
LISTALL
