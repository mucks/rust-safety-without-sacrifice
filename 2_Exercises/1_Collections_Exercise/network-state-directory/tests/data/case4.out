ALLIANCES: ["ForestCity â†” Zuitzerland"]
LISTALL total=2
 - Zoe#10 (Citizen)
 - Pax#11 (Founder)
